http-server/apache2-web-server/README.md

```diff
@@ -82,31 +101,31 @@ $ nano web/htdocs/index.html
 </html>
 ```
 
-Or Download it from this repo:
+Or download it from this repo:
 
 ```
-$ wget https://raw.githubusercontent.com/sylabs/examples/master/http-server/apache2-web-server/index.html
+$ wget -O httpd/htdocs/index.html https://raw.githubusercontent.com/sylabs/examples/master/http-server/apache2-web-server/index.html
 ```
 
-
 <br>
 
-
-To use this structure, we start up an instance binding our host path into the container:
+Now our directory map should look like this:
 
 ```
-$ singularity instance start \
- -B web/htdocs:/usr/local/apache2/htdocs \
- -B web/logs:/usr/local/apache2/logs \
- httpd.sif httpd
+~/httpd/
+ |-- htdocs/
+ |   `-- index.html
+ `-- logs/
 ```
 
 <br>
 
-*FYI:* the above command is the same as:
+## Start the instance:
+
+To use this structure, we start up an instance binding our host path into the container:
 
 ```
-$ singularity instance start -B web/htdocs:/usr/local/apache2/htdocs -B web/logs:/usr/local/apache2/logs httpd.sif httpd
+$ singularity instance start -B httpd/:/srv/httpd/ httpd.sif httpd
```

Comment:
If we use the '`~/`' (home directory), then it should be:

```
$ singularity instance start -B ~/httpd/httpd/:/srv/httpd/ httpd.sif httpd
```

But since we are in the `~/httpd/` directory, we don't need that part. (I think...)

Besides, I don't want the user thinking there limited to `~/`.